repos/system_upgrade/el9toel10/actors/nonscarhsm/libraries/nonscarhsm.py

```diff
@@ -0,0 +1,32 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import RHSMInfo
+from leapp.reporting import create_report
+
+
+def process():
+    for info in api.consume(RHSMInfo):
+        if info.is_registered and not info.sca_detected:
+            # TODO summary, remediation hint, external link
+            create_report(
+                [
+                    reporting.Title(
+                        "The system is not registered to an SCA organization"
```

Comment:
```suggestion
                        "The system is registered to an account with disabled Simple Content Access (SCA)"
```

Reply:
@ptoscano wdyt?

Reply:
hmm not convinced about "disabled SCA"; we have feedback about the usage of "disabled" in the context of SCA when using `subscription-manager status`, so I'd like to avoid mixing the two together again